internal/armadactl/reprioritize.go

```diff
@@ -76,7 +74,7 @@ func (a *App) writeResults(results map[string]string) error {
 	}
 
 	if len(erroredIds) > 0 {
-		return fmt.Errorf("[armadactl.writeResults] error reprioritizing some jobs")
+		return fmt.Errorf("error reprioritizing some jobs")
```

Comment:
I think it is always better to use `errors.Errorf` as from the `github.com/pkg/errors` as they include stack traces opposed to `fmt` package.

Reply:
Done